cheatsheets/JWT_Cheat_Sheet.md

```diff
@@ -0,0 +1,227 @@
+# JWT Cheat Sheet
+
+## Introduction
+
+Many applications use **JSON Web Tokens** (JWT) to allow the client to indicate its identity for further exchange after
+ authentication and to securely transmit data.
+
+From [JWT.IO](https://jwt.io/introduction):
+
+> JSON Web Token (JWT) is an open standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) that defines a compact and
+> self-contained way for securely transmitting information between parties as a JSON object. This information can be
+> verified and trusted because it is digitally signed. JWTs can be signed using a secret (with the **HMAC** algorithm)
+> or a public/private key pair using **RSA** or **ECDSA**.
+
+JSON Web Token is used to carry information related to the identity and characteristics (claims) of a client. This
+ information should signed by the server in order for it to detect whether it was tampered with after sending it to the
+ client. This will prevent an attacker from changing the identity or any characteristics (for example, changing the role
+ from simple user to admin or change the client login).
+
+This token is created during authentication (is provided in case of successful authentication) and is verified by the
+ server before any processing. It is used by an application to allow a client to present a token representing the user's
+ "identity card" to the server and allow the server to verify the validity and integrity of the token in a secure way,
+ all of this in a stateless and portable approach (portable in the way that client and server technologies can be
+ different including also the transport channel even if HTTP is the most often used).
+
+
+### Verification
+
+Your application, absolutely should verify that the tokens it recieves are valid. You can do this with a symmetric scheme, where
+a secret between all parties out of band. For more on how to manage secrets please refer to the [secrets management cheat sheet](Secrets_Management_Cheat_Sheet.md). Or you can use an asymmetric key to signe your tokens.  you have the option of signing your
```

Comment:
> a [shared] *secret* between *all parties* out of band

It is dangerous to share a secret between more than two parties. I would say "between *two* parties".